front/app/containers/SiteMap/index.tsx

```diff
@@ -118,6 +121,8 @@ interface Props extends DataProps {}
 
 const SiteMap = ({ projects, authUser }: Props) => {
   const loaded = projects !== undefined;
```

Comment:
Since the 'About' and 'FAQ' pages can now be deleted, they should not be hardcoded anymore in the SiteMap.

Reply:
I didn't think of that, nice catch. Actually, we should probably add all the pages available from the navbar to this sitemap as well

Reply:
Added this to my todo list :+1: